lab-jason/lib/joke.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+
+function tellJoke(filename, cb){
+  fs.readFile(`./assets/${filename}`, (err, buff) => {
+    if (err) {
+      cb(err);
+    }
+    if(cb) { cb(null, buff.toString('hex', 0, 8)); }
+  });
+}
+
+tellJoke('one.txt',
+  () => tellJoke('two.txt',
+    () => tellJoke('three.txt', (err) => {
+      if (err) throw err;
+    })
+  )
+);
+
+exports.readJoke = function(file1, file2, file3, cb) {
+  var datSmash = '';
+  tellJoke(file1, (err, data1) => {
+    if (err) { cb(err); }
+    if (cb) {datSmash += data1;}
+
+    tellJoke(file2, (err, data2) => {
+      if (err) { cb(err); }
+
+      if (cb) {datSmash += data2;}
+
+      tellJoke(file3, (err, data3) => {
+        if (err) { cb(err); }
+        datSmash += data3;
+        if (cb) {cb(err, datSmash);}
```

Comment:
I'm pretty sure your test is failing for two related reasons:

1. You're not returning with the `cb(err)`, so the code keeps going.
2. You're passing `err` here instead of `null`.